src/backend/src/bitcoin_utils.rs

```diff
@@ -0,0 +1,20 @@
+use bitcoin::{Address, Network, PublicKey};
+use ic_cdk::api::management_canister::bitcoin::BitcoinNetwork;
+
+fn transform_network(network: BitcoinNetwork) -> Network {
```

Comment:
NIT: Since this is a 1-1 match, why not implement it as
```rust
impl From<BitcoinNetwork> for Network
```

Reply:
good suggestion!

Reply:
ah wait, `Network` is not in my crate, so I cannot `impl` it

Reply:
however, it could be useful to have this in the `ic_cdk`

```rust
impl BitcoinNetwork {
    pub fn to_network(self) -> Network {
        match self {
            BitcoinNetwork::Mainnet => Network::Bitcoin,
            BitcoinNetwork::Testnet => Network::Testnet,
            BitcoinNetwork::Regtest => Network::Regtest,
        }
    }
}


```